tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -111,6 +111,18 @@ Verify Pytorch And Tensorflow Can Be Spawned
     Verify Image Can Be Spawned  image=pytorch  size=Default
     Verify Image Can Be Spawned  image=tensorflow  size=Default
 
+Verify That Blackbox-exporter Is Protected With Auth-proxy
+    [Documentation]     Verify that Blackbox-exporter is protected with auth-proxy and check it has 2 containers
+    [Tags]  Sanity
+    ...     Tier1
+    ...     ODS-1090
+    ${pod} =    Find First Pod By Name    namespace=redhat-ods-monitoring    pod_start_with=blackbox-exporter-
+    @{containers} =    Get List Of Container    pod_name=${pod}    namespace=redhat-ods-monitoring
+    List Should Contain Value    ${containers}    oauth-proxy
+    List Should Contain Value    ${containers}    blackbox-exporter
+    @{links} =    Get Endpoint URLs From Targets    target_name=user_facing_endpoints_status
+    Length Should Be    ${links}    2
+    Verify Link And Run Commands On Prometheus Pod For Jupyterhub And Rhods-Dashboard URLs    links=${links}
```

Comment:
You could rename this to `Verify Authentication Is Required To Access BlackboxExporter` and get the target links inside it (no need to be a parameter in my opinion)